### ISSUE TITLE
can't install after manual delete lux-wine folder

### ISSUE THREAD

**Issue by vanbwodonk** [VHSgunzo/lux-wine#24]
I used arch linux, install command get this error:
```
❯ curl -sL lwrap.github.io|bash
[ INFO ][2024.04.19 15:36:38]: Found RunImage internal config: Run.rcfg 
[ INFO ][2024.04.19 15:36:38]: Remounting RunImage with squashfuse... 
[ INFO ][2024.04.19 15:36:38]: Autorun mode: lwrap 
[ WARNING ][2024.04.19 15:36:38]: The HOSTEXEC option is enabled! 
[ INFO ][2024.04.19 15:36:38]: Installing Lux Wine... 
[ ERROR ][2024.04.19 15:36:40]: Failed to install Lux Wine! 

```
How to clean uninstall? and then install again?

**Comment by VHSgunzo** [comment]
@vanbwodonk 
Hi, tell me which directory exactly did you delete?

The output of the command shows that you still have the lwrun container. In that case, just try reinstalling the LW scripts:
```
lwrun --install
```

---

In the future, if you want to [uninstall LW](https://github.com/VHSgunzo/lux-wine?tab=readme-ov-file#removing), use the command:
```
lwrun --uninstall
```

**Comment by vanbwodonk** [comment]
>  Hi, tell me which directory exactly did you delete?
/home/user/.local/share/LutrisWine

After delete this my error message is changed
```
[ INFO ][2024.04.23 08:38:15]: Update lwrap... 
mv: cannot stat '/home/drw/.local/share/LuxWine/bin/lwrap.new12610': No such file or directory
chmod: cannot access '/home/drw/.local/share/LuxWine/bin/lwrap': No such file or directory
[ INFO ][2024.04.23 08:38:16]: Downloading lwrun... 
[ INFO ][2024.04.23 08:38:20]: lwrun downloaded! 
bash: line 977: /home/drw/.local/share/LuxWine/bin/lwrun: No such file or directory
```

**Comment by vanbwodonk** [comment]
yes, i used since old lutris-wine,
tried deleting your suggesting folder, still get this error:

```
❯ curl -sL lwrap.github.io|bash
[ INFO ][2024.04.24 08:22:22]: Update lwrap... 
mv: cannot stat '/home/drw/.local/share/LuxWine/bin/lwrap.new6643': No such file or directory
chmod: cannot access '/home/drw/.local/share/LuxWine/bin/lwrap': No such file or directory
[ INFO ][2024.04.24 08:22:23]: Downloading lwrun... 
[ INFO ][2024.04.24 08:22:28]: lwrun downloaded! 
bash: line 977: /home/drw/.local/share/LuxWine/bin/lwrun: No such file or directory
```

**Comment by vanbwodonk** [comment]
I found the problem. This is all because `lwrap.github.io` script using my `aria2.conf` inside  `~/.aria2/aria2.conf`. 
Because I'm using modified aria2.conf. The script failed to download requirement files.
I think in the future the script can use aria2 command with `--no-conf` option.

**Comment by VHSgunzo** [comment]
> I think in the future the script can use aria2 command with `--no-conf` option.

Interesting. It has also recently been revealed that wget options are slightly incompatible with wget2. I will add these fixes in the next release. 

In the meantime, you can install LW by downloading the lwrun container, make it executable and run it.
https://github.com/VHSgunzo/lux-wine?tab=readme-ov-file#offline-installation

**Comment by VHSgunzo** [comment]
fixed in [v0.78.4.6](https://github.com/VHSgunzo/lux-wine/releases/tag/v0.78.4.6)
